django/db/models/functions/datetime.py

```diff
@@ -87,6 +87,13 @@ class ExtractDay(Extract):
     lookup_name = 'day'
 
 
+class ExtractWeek(Extract):
+    """
+    Returns 1-53 based on ISO-8601, i.e., Monday is the first of the week.
```

Comment:
1-52 or 1-53. For example 2017 will have only 52 weeks (based on ISO). IMO it should be clear, that week 53th doesn't always occur.